var/spack/repos/builtin/packages/aomp/package.py

```diff
@@ -247,7 +319,7 @@ def patch(self):
             kwargs = {'ignore_absent': False, 'backup': False, 'string': False}
             with working_dir('aomp-dir/hip-on-vdi'):
                 match = '^#!/usr/bin/python'
-                python = self.spec['python'].command.path
+                python = '/usr/bin/env python2.7'
```

Comment:
Can you explain this change? It looks like a step in the wrong direction to me.

Reply:
This was done by mistake. reverted